tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/context/BMainInstance.java

```diff
@@ -292,6 +292,51 @@ public void runMain(String command, String[] args, Map<String, String> envProper
         }
     }
 
+
+    /**
+     * Executing the sh or bat file to start the server and return the PID for service handling.
+     *
+     * @param command       command to run
+     * @param args          command line arguments to pass when executing the sh or bat file
+     * @param envProperties environmental properties to be appended to the environment
```

Comment:
```suggestion
     * @param envProperties environment properties
```
minor suggestion, can be fixed later